src/test/java/edu/kis/powp/jobs2d/VisitorTest.java

```diff
@@ -0,0 +1,24 @@
+package edu.kis.powp.jobs2d;
+
+import edu.kis.powp.jobs2d.command.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class VisitorTest {
+	public static void main(String[] args) {
+		List<DriverCommand> commands = new ArrayList<>();
+
+		commands.add(new SetPositionCommand(-50, -50));
+		commands.add(new OperateToCommand(-50, 50));
+		commands.add(new OperateToCommand(50, 50));
+		commands.add(new OperateToCommand(50, -50));
+		commands.add(new OperateToCommand(-50, -50));
+
+		CommandVisitor commandCounter = new CommandCounter();
+
+		for(DriverCommand command : commands){
+			command.accept(commandCounter);
+		}
+	}
```

Comment:
OK, but we need to add the test to the GUI application, e.g., the counter test should take the current command (from using commands manager) and write in the log stats concerning number of inner OperateTo and SetPosition command